silkrpc/commands/trace_api.cpp

```diff
@@ -79,12 +79,33 @@ asio::awaitable<void> TraceRpcApi::handle_trace_call(const nlohmann::json& reque
 
 // https://eth.wiki/json-rpc/API#trace_callmany
 asio::awaitable<void> TraceRpcApi::handle_trace_call_many(const nlohmann::json& request, nlohmann::json& reply) {
+    auto params = request["params"];
+    if (params.size() < 2) {
+        auto error_msg = "invalid trace_callMany params: " + params.dump();
+        SILKRPC_ERROR << error_msg << "\n";
+        reply = make_json_error(request["id"], 100, error_msg);
+        co_return;
+    }
+    const auto trace_calls = params[0].get<std::vector<trace::TraceCall>>();
+    const auto block_number_or_hash = params[1].get<BlockNumberOrHash>();
+
+    SILKRPC_INFO << "#trace_calls: " << trace_calls.size() << " block_number_or_hash: " << block_number_or_hash << "\n";
```

Comment:
I would demote this log line to `DEBUG`

Reply:
It was done following the way of all other methods that use `INFO` to show parameters...